internal/acctest/config/oauth/authserversettings/oauth_auth_server_settings_test.go

```diff
@@ -83,6 +79,25 @@ func TestAccOauthAuthServerSettings(t *testing.T) {
 }
 
 func testAccOauthAuthServerSettings(resourceName string, resourceModel oauthAuthServerSettingsResourceModel, includeAllAttributes bool) string {
+	addUpdatedResourceModelFields := []string{}
+	if resourceModel.bypassActivationCodeConfirmation == true {
+		addUpdatedResourceModelFields = append(addUpdatedResourceModelFields, "bypass_activation_code_confirmation = true")
```

Comment:
Seems like these values should come from the resourceModel rather than being hard-coded

Reply:
this is now updated